app/src/actioncommands.cpp

```diff
@@ -713,9 +713,11 @@ void ActionCommands::moveFrameForward()
     Layer* layer = mEditor->layers()->currentLayer();
     if (layer)
     {
-        if (layer->moveKeyFrameForward(mEditor->currentFrame()))
+        mEditor->notifyFrameMoved();
```

Comment:
I've reworked this, it made no sense to notify that the cache should be updated from actioncommands (nor anywhere else), instead I think it's up to the model to tell that there are frames to be updated, so that's the approach I've taken.